### PR TITLE
6433 readspeaker web reader implementation

### PR DESCRIPTION
**Pull request by TeroOtavia**
Readspeaker webReader is now implemented to Muikku. It is only currently used in the workspace materials. For this use case, everything in the materials like material loader is configurated to work with webReader. Most of the material fields are skipped and so on...

Resolves: #6433

### CODE REVIEW

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/select-field.tsx [hunk 1]
```diff
@@ -226,44 +227,53 @@ export default class SelectField extends React.Component<
...
-              size={
-                this.props.content.listType === "list"
-                  ? this.props.content.options.length
-                  : null
-              }
-              disabled
-            />
-          </span>
+          <>
+            <ReadspeakerMessage text="Valintakenttä" />
```
Comment (avukkonen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/select-field.tsx:231]: TODO: lokaalit

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/select-field.tsx [hunk 2]
```diff
@@ -351,91 +361,97 @@ export default class SelectField extends React.Component<
       const selectFieldType =
         this.props.content.listType === "list" ? "list" : "dropdown";
       return (
+        <>
+          <ReadspeakerMessage text="Valintakenttä" />
```
Comment (avukkonen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/select-field.tsx:365]: TODO: lokaalit

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/text-field.tsx [hunk 3]
```diff
@@ -445,27 +464,30 @@ export default class TextField extends React.Component<
...
-          >
-            {component}
-          </Dropdown>
-        ) : (
-          component
-        )}
-        {correctAnswersummaryComponent}
-      </span>
+      <>
+        <ReadspeakerMessage text="Tekstikenttä" />
```
Comment (avukkonen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/text-field.tsx:468]: TODO: localization

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/static/mathjax.tsx [hunk 4]
```diff
@@ -47,17 +48,25 @@ export default class MathJAX extends React.Component<
     }
     //TODO remove the data-muikku-word-definition thing, it's basically used for styling alone
     if (this.props.invisible) {
-      return <span className="math-tex">{this.props.children}</span>;
+      return (
+        <>
+          <ReadspeakerMessage text="Matikkakaava" />
```
Comment (avukkonen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/static/mathjax.tsx:53]: TODO : localization

**Review by avukkonen — CHANGES_REQUESTED** [review]

**Review by avukkonen — DISMISSED** [review]

**Review by avukkonen — DISMISSED** [review]

**Review by avukkonen — APPROVED** [review]

**Review by jjlankinen — APPROVED** [review]
Questionable approve but PR is tested quite well. Further testing will be done in demo server after the merge.

**Pull request merged**